metrics/prometheus/prometheus.go

```diff
@@ -1,196 +1,202 @@
-// (c) 2021, Ava Labs, Inc. All rights reserved.
+// (c) 2021-2025, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
 package prometheus
 
 import (
+	"errors"
+	"fmt"
 	"sort"
 	"strings"
 
-	"github.com/ava-labs/subnet-evm/metrics"
-
 	"github.com/prometheus/client_golang/prometheus"
 
+	"github.com/ethereum/go-ethereum/metrics"
+
 	dto "github.com/prometheus/client_model/go"
```

Comment:
Any idea why `dto` is the standard way to alias this package?

Reply:
It's for data transfer object... Package is `io_prometheus_client` in a `go` directory, so not too sure how it should be named 😄